core/codesafe.cmake

```diff
@@ -26,7 +26,7 @@ set(CMAKE_C_COMPILER ${CODESAFE_GCC})
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mpowerpc -mcpu=e5500 -mno-toc -mbig-endian -mhard-float -mno-multiple -mno-string -meabi -mprototype -mstrict-align -memb -fno-builtin -DNF_CROSSCC=1 -DNFAST_CONF_H='\"nfast-conf-auto.h\"' -DNF_CROSSCC_PPC_GCC=1 -D_THREAD_SAFE -DBYTE_ORDER=4321")
 
 set(CMAKE_C_LINK_EXECUTABLE
-    "${CODESAFE_GCC} -Wl,-wrap=read -Wl,-wrap=write -Wl,-wrap=socket -Wl,-wrap=send  -Wl,-wrap=sendto -Wl,-wrap=recv -Wl,-wrap=recvfrom -Wl,-wrap=listen -Wl,-wrap=connect -Wl,-wrap=bind -Wl,-wrap=setsockopt -Wl,-wrap=select -Wl,-wrap=accept -Wl,-wrap=poll -Bstatic -o <TARGET> <OBJECTS> <LINK_LIBRARIES> ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/hoststdioeinetsocks.o ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libfaksys.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libcutils.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libvfsextras.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libseewrpr.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libipccore.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libsolotrace.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libnfstub.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libfaksys.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libseewrpr.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/seelib.a ${NFAST_PATH}/c/csd/lib-ppc-linux-gcc/libipccore.a -lpthread -lrt")
+    "${CODESAFE_GCC} -Wl,-wrap=read -Wl,-wrap=write -Wl,-wrap=socket -Wl,-wrap=send  -Wl,-wrap=sendto -Wl,-wrap=recv -Wl,-wrap=recvfrom -Wl,-wrap=listen -Wl,-wrap=connect -Wl,-wrap=bind -Wl,-wrap=setsockopt -Wl,-wrap=select -Wl,-wrap=accept -Wl,-wrap=poll -Bstatic -o <TARGET> <OBJECTS> <LINK_LIBRARIES> ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/objects-Debug/hoststdioeinetsocks/vfsextras/hoststdioeinetsocks.c.o ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libfaksys.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libcutils.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libvfsextras.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libseewrpr.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libipccore.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libsolotrace.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libnfstub.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libfaksys.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libseewrpr.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/seelib.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libipccore.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libnflog.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libcutils.a ${NFAST_PATH}/c/csd/lib-glibc-ppc-gcc/libgsexamples.a -lpthread -lrt")
```

Comment:
you sure we need libgsexamples.a?

Reply:
Unfortunately yes, `libgsexamples.a` provides a number of symbols referenced by `/opt/nfast/c/csd/lib-glibc-ppc-gcc/libvfsextras.a`.

Removing `libgsexamples.a` results in undefined references to `sbn_bignumreceiveupcall`, `sbn_bignumsendlenupcall`, `sbn_bignumsendupcall`, `sbn_bignumformatupcall`.